dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -663,7 +663,7 @@ class ScriptsControllerTest < ActionController::TestCase
 
     get :show, params: {id: 'test-fixture-visible-after'}
     assert_response :success
-    assert response.body.include? 'The lesson Stage 1 will be visible after'
+    assert response.body.include? 'The lesson stage 1 will be visible after'
```

Comment:
Should this be updated to Lesson or not changed?

Reply:
Yea I was a little careless with these, haha. I can fix these to Lesson

Reply:
On second thought, I'd like to leave these as-is until we follow up with the work to change the script DSL format. It's not ideal that I renamed them uselessly but I think we'll be changing them again soon so I'd rather not change it again.

Reply:
Ok thats fine